<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="LoggingTab">
     <attribute name="title">
      <string>Logging</string>
     </attribute>
     <widget class="QListView" name="listView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>461</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="HaltDialogTab">
     <attribute name="title">
      <string>HaltDialog</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="ResultTab">
     <attribute name="title">
      <string>Result</string>
     </attribute>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>441</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOpenCam">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>90</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>openCam</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>391</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>600</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEnde">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ENDE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
